.circleci/config.yml

```diff
@@ -32,7 +32,22 @@ workflows:
     when:
       not: << pipeline.parameters.mapbox_navigation_native_upstream >>
     jobs:
-      - release-snapshot
+      - release-snapshot-start:
+          type: approval
+          filters:
+            branches:
+              ignore:
+                - main
+                - /release-v.*/
+      - release-snapshot:
+          requires:
+            - release-snapshot-start
+      - release-snapshot:
```

Comment:
Why do you have 2 jobs with the same name?

Reply:
This is one job. We run it by:
- main and release branches
- approval in circle ci
- set PR label